lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -71,6 +71,24 @@
     default: TLSv1
```

Comment:
When the new iLO version is installed, this argument is not used. I think you should hint at this in the docs or use this argument to create the ssl_context in the new version. If I set ssl_version to TLSv1_2 I expect that a TLSv1_1 connection would fail, but that is not the case with this implementation.

Reply:
I could also add a noop `ssl_version` parameter to python_hpilo for backwards compatibility. Though as you're already writing 4.0+ specific code to handle certificate verification, I'm wondering if that would actually even be useful for you.